arak.example.toml

```diff
@@ -5,6 +5,10 @@ ethrpc = "http://localhost:8545"
 [database.sqlite]
 connection = "file:arak.db"
 
+#[database.postgres]
```

Comment:
Why not delete this?

Reply:
Because the example file doesn't give an example of postgres (which is our primary purpose). Just wanted the potential user to be aware of how that it configured.